utils/replay_record/Makefile

```diff
@@ -0,0 +1,8 @@
+all:
```

Comment:
This directory needs a short README.md what it actually does, even if it just points the the README.persistent_mode.md file.